test/examples/correct/brochu_2003.json

```diff
@@ -1,5 +1,5 @@
 {
-    "@context": "../../docs/context/v0.3.0/phyx.json",
+    "@context": "../../../docs/context/development/phyx.json",
```

Comment:
Did you mean to introduce this change? And if so, can you explain why?

Reply:
Yup, this is the "Also fixes the `@context` path in brochu_2003.json" note in the PR. Two things needed to be changed here:
1. We need to add the `../` because this file is now inside the `correct` directory.
2. We previously referred to the context directory as "v0.3.0", but I changed that in a previous PR to "development" so that it wouldn't be published as v0.3.0 until we had finalized that. Unfortunately, I didn't catch that until I started testing with JPhyloRef in PR #62. This fix (along with the [fix to the part of the testing script that loaded the JSON-LD context](https://github.com/phyloref/phyx.js/pull/67/files#diff-ec2e836dc27031f9e239e86551e13f353e1c7969de3f39412bb1cf744ac4f5ecR70-R76)) ended up here during my PR reorganization today, but I could try to move them to PR #62 if that makes more sense?

Reply:
I'm not sure I'm following, but you answered my question with reasonable arguments, so I'm going to resolve this.